CHANGELOG.rst

```diff
@@ -13,6 +13,19 @@ Change Log
 
 Unreleased
 ~~~~~~~~~~
+
+[2.0.0] - 2020-01-08
+~~~~~~~~~~~~~~~~~~~~
+* Switch to using edx-django-utils TieredCache (a two-layer cache that uses both
+  Django's cache and an internal request-level cache) to reduce the number of
+  memcached roundtrips. This was a major performance issue that accounted for
+  10-20% of transaction time for certain courseware views in edx-platform.
+* It is now REQUIRED to [add `RequestCacheMiddleware` to middleware](https://github.com/edx/edx-django-utils/tree/master/edx_django_utils/cache#tieredcachemiddleware).
```

Comment:
This looks like a Markdown link, but this is an rST file.

Reply:
lol, sorry. GitHub habits.